ansible/site.yml

```diff
@@ -1,5 +1,34 @@
 ---
+
+# Note: OS specific variables are included in
+# the order of the hierarchy of their fork...
+# The "lower" fork wins
+# Debian <
+#  Ubuntu <
+#   Raspbian
+#   Armian
+
 - hosts: all
+  pre_tasks:
+    - name: Include OS family variables
+      include_vars: "group_vars/{{ ansible_os_family|lower }}"
+    - name: Include Distribution variables
+      include_vars: "group_vars/{{ ansible_distribution|lower }}"
```

Comment:
You are correct, but I think we should keep it so that we have options for future development. Variables for ansible_os_family = Debian may not always be the same as ansible_distribution = Ubuntu.